doc/transformation.md

```diff
@@ -0,0 +1,24 @@
+#Transformation Writeup
+
+I wonder what this really is... [enc](https://mercury.picoctf.net/static/1d8a5a2779c4dc24999f0358d7a1a786/enc) \
+`''.join([chr((ord(flag[i]) << 8) + ord(flag[i + 1])) for i in range(0, len(flag), 2)])`
+
+Downloading and opening the enc file we see "灩捯䍔䙻ㄶ形楴獟楮獴㌴摟潦弸彥㜰㍢㐸㙽". The name of the file is enc so this is
+probably the encrypted flag. Looking back at the problem, we can see a little code snippet. The variable name that was
+used was flag, so it most likely would be how the flag was encrypted. To reverse what they did to encrypt the flag, we
+need to first understand what is being done. We can break the code up into 3 parts.
+1. `''.join([ ])`
+2. `chr( ) for i in range(0, len(flag), 2)`
+3. `ord(flag[i] << 8) + ord(flag[i + 1])`
+
+The first part is formatting the answer so that there are no spaces in the encrypted flag. The second part is iterating
```

Comment:
The first part is not for "formatting" but for concatenating a list of characters into a string, and use `''` as separator (i.e. no separator)

Reply:
fixed